src/xtgeo/cube/cube1.py

```diff
@@ -919,8 +782,15 @@ def to_file(self, sfile, fformat="segy", pristine=False, engine="xtgeo"):
 
         fobj.check_folder(raiseerror=OSError)
 
+        if engine is not None:
+            warnings.warn(
+                "Providing an 'engine' value is no longer supported and will have no "
+                "effect in the future. segyio will eventually be used by default.",
+                UserWarning,
+            )
+
         if fformat in FileFormat.SEGY.value:
-            _cube_export.export_segy(self, fobj.name, pristine=pristine, engine=engine)
+            _cube_export.export_segy(self, fobj.name, pristine=pristine)
```

Comment:
Maintains the default engine being "xtgeo" until the segyio engine is fully tested. `export_segy()` holds this default as a kwarg.